### ISSUE TITLE
URL Lists

### ISSUE THREAD

**Issue by ieee8023** [Darkhogg/LWJTorrent#3]
Do you have any plans to support URL lists? 

I believe GetRight outlined how it works:
http://getright.com/seedtorrent.html

**Comment by Darkhogg** [comment]
Yes, that's something that will be supported. But I can't tell _when_, as currently there is nothing _working_ as an actual torrent client.
